docs/getting-started.md

```diff
@@ -0,0 +1,80 @@
+# Prerequisites
+
+- [We recommend using VOLTA to manage your JavaScript command-line tools](volta.sh)
+- Node.js® version 14.18.1
+- Yarn - Please use Yarn v1 instead of npm to install dependencies  
+  `npm install --global yarn@1.22.17`
+
+## Visual Studio Code - recommended extensions
+
+- Angular Language Service
+- Prettier - Code formatter
+- Nx Console
+- EditorConfig for VS Code
+- markdownlint
+- ESLint
```

Comment:
suggestion: Can we add these extensions to the `extensions.json` file under `.vscode` folder in the repo so VS Code prompts the developer to install them?

Reply:
They are added now, should we just remove this section?

Reply:
Yes, I think so. Can be replaced with a sentence saying that the extensions exist and how to install them (I think there's a command in VS Code that can install the recommended extensions if the file exists).